more/free-programming-cheatsheets.md

```diff
@@ -105,6 +106,9 @@
 
 * [Vue Essential Cheatsheet](https://www.vuemastery.com/pdf/Vue-Essentials-Cheat-Sheet.pdf) - Vue Mastery (PDF)
 
+#### React.js
```

Comment:
```suggestion

#### React.js

```

There weren't the [correct number of black lines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#formatting).

Reply:
Resolved